spec/unit/actions/synchronize_user_filters_spec.rb

```diff
@@ -44,13 +44,8 @@
         allow(File).to receive(:open).and_return("project_id\n123456789")
       end
 
-      it 'uses project_id column' do
-        expect(GoodData::UserFilterBuilder).to receive(:get_filters) do |filters, _|
-          CSV.foreach(filters, headers: true, return_headers: false) do |row|
-            expect(row['project_id']).to eq '123456789'
-          end
-          []
-        end
+      it 'uses client_id column' do
```

Comment:
Identical code found in synchronize_user_filters_spec.rb:48, synchronize_users_spec.rb:47 (mass = 64)